data/examples/declaration/value/function/infix/op-chain-r-eq-dollar-2-out.hs

```diff
@@ -1,11 +1,11 @@
 -- Right chain, $ case, 2 operators with p(a) == p($)
 p :: Int
 p =
-  1 `seq`
-    2 `seq`
-      3 `seq`
-        4 $
-          5 `seq`
-            6 `seq`
-              7 `seq`
-                8
+  1
+    `seq` 2
+    `seq` 3
+    `seq` 4
+    $ 5
+      `seq` 6
+      `seq` 7
+      `seq` 8
```

Comment:
`seq` is available in Base: https://hackage.haskell.org/package/base-4.17.0.0/docs/doc-index-S.html and has `infixr 0` fixity, so this example shouldn't change IMO.

Reply:
Yeah, I already noticed this. Will push a fix shortly.